kits/outage-detector/prompts/outage-detector_InstructorLLMNode_837_user.md

```
@@ -0,0 +1,4 @@
+Suspected component: {{InstructorLLMNode_311.output.suspected_component}}
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)